index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" type="text/css">
+    <script src="https://kit.fontawesome.com/5d13e94c8b.js" crossorigin="anonymous"></script>
+</head>
+<body>
+   <header class="header">
+       <a href="#"><h1>Kayonga Chiteta</h1></a>   
+       
+       <nav class="nav"><i class="fas fa-bars two"></i> </nav>
+
+    </header>
+        <div class="container">
```

Comment:
- [ ] The container of the section is a `section` tag. Let's change this, As requested by the previous reviewer also 💪